src/components/Cart/CartItemStyles.ts

```diff
@@ -10,10 +10,20 @@ const StyledCartItem = styled(Item)`
   align-items: center;
   justify-content: space-between;
 
+  //TODO:
+  @media (max-width: 540px) {
+    /* GRID MOBILE*/
+  }
+
   & .cart-item__title {
     font-size: 1.5rem;
     text-align: left;
     width: 40%;
+
+    @media (max-width: 540px) {
```

Comment:
it's better when all the media queries are consistent and coming from your theme